pkg/cloud/azure/actuators/machine/actuator_test.go

```diff
@@ -140,7 +140,7 @@ func newFakeScope(t *testing.T, label string) *actuators.MachineScope {
 	}
 	scope.Network().APIServerIP.DNSName = "DummyDNSName"
 	labels := make(map[string]string)
-	labels["set"] = label
+	labels["machine.openshift.io/cluster-api-machine-role"] = label
```

Comment:
I see `"machine.openshift.io/cluster-api-machine-role"` used ~4 times - turn it into a const?

Reply:
Thanks fixed.